sklearn/calibration.py

```diff
@@ -311,145 +370,202 @@ def _more_tags(self):
         }
 
 
-class _CalibratedClassifier:
-    """Probability calibration with isotonic regression or sigmoid.
+def get_prediction_method(clf_fitted, return_string=False):
```

Comment:
Let's keep this explicitly private, even though the file is private as well.

Reply:
Also as mentioned above let's remove `return_string` and always return the method object, since we can just rely on the `__name__ attribute.